packages/legend-application-studio/src/stores/project-viewer/ProjectViewerStore.ts

```diff
@@ -472,33 +472,19 @@ export class ProjectViewerStore {
         this.editorStore.applicationStore.notifyError(
           `Can't build graph. Redirected to text mode for debugging. Error: ${error.message}`,
         );
-        try {
-          const editorGrammar =
-            (yield this.editorStore.graphManagerState.graphManager.entitiesToPureCode(
-              entities,
-            )) as string;
-          yield flowResult(
-            this.editorStore.grammarTextEditorState.setGraphGrammarText(
-              editorGrammar,
-            ),
-          );
-        } catch {
+        yield flowResult(
```

Comment:
note, here we're making assumption we're already in form mode, what if we're not?

Reply:
this function gets called only when we initialize studio. So the mode we are in would be FORM